src/PowerSystemCaseBuilder.jl

```diff
@@ -84,6 +87,11 @@ Category for PowerSimulationsDynamics.jl examples.
 """
 struct PSIDSystems <: SystemCategory end
 
+"""
+Category for SiennaPRASInterface.jl examples.
+"""
+struct SPISystems <: SystemCategory end
```

Comment:
We should probably make this something more generic like RASystems.

Reply:
I agree. This just follows convention we used for other packages.